src/SprykerEco/Yves/CrefoPay/Theme/default/components/atoms/crefopay-checkbox-helper/crefopay-checkbox-helper.ts

```diff
@@ -0,0 +1,67 @@
+import Component from 'ShopUi/models/component';
+
+export default class CrefopayCheckboxHelper extends Component {
+
+    protected paymentForm: HTMLElement;
+    protected triggers: HTMLElement[];
+    protected triggerInput: any;
+    protected targets: HTMLInputElement[];
+
+    protected readyCallback(): void {
+        this.paymentForm = <HTMLElement>document.querySelector(this.containerSelector);
+        this.triggers = <HTMLElement[]>Array.from(this.paymentForm.querySelectorAll(this.triggerSelector));
+        this.targets = <HTMLInputElement[]>Array.from(this.paymentForm.querySelectorAll(this.targetSelector));
+
+        this.applyAttributes();
+        this.mapEvents();
+    }
+
+    protected mapEvents(): void {
+        this.triggers.forEach((trigger: HTMLElement) => {
+            trigger.addEventListener('change', (event: Event) => this.onChange(event));
+        });
+    }
+
+    protected applyAttributes (): void {
+        this.targets.forEach((target: HTMLInputElement)=>{
+            target.setAttribute(this.customAttrName, this.customAttrValue);
+        })
+    }
+
+    protected onChange(event: Event): void {
+        this.targets.forEach((target: HTMLInputElement)=>{
+            target.checked = false;
+        });
+        this.syncCheckboxes(event.currentTarget);
+    }
+
+    protected syncCheckboxes(triggered: EventTarget): void {
```

Comment:
Why the name is `syncCheckboxes`? It is working only with one checkbox. And it not syncs but makes it checked. Suggest to rename into `checkCheckbox`.
`triggered`-> `checkbox`.